### ISSUE TITLE
Do I need to change any model settings for different provided models?

### ISSUE THREAD

**Issue by Cugtyt** [xinntao/EDVR#66]
If I test different models in model zoo, do I need to change some code or settings?

**Comment by xinntao** [comment]
Just change the `data_mode = 'sharp_bicubic'` in `test_Vid4_REDS4_with_GT.py` is OK. 
The testing codes will do the modifications.

**Comment by Cugtyt** [comment]
During training, how to use the deblur module? I notice that there is a deblur option, is that just turn it on is ok?

**Comment by xinntao** [comment]
The only difference between SR and deblur is that they have different input size. 
So you only need to change `HR_in` from False to True.

For deblurring, it is better to have a predeblur model before alignment and fusion. So You can also turn the `predeblur` into True. But in our released model (used in the competition), we first pre-train this predublur module and then train the whole model. If you only set `predeblur: True` and do not have a pretraining, it may reduce performance.

**Comment by Cugtyt** [comment]
Could you please provide more information about the predeblur mudule training? I tried to add conv layers after predeblur to reconstruct clean image and train with the whole model, but didn't see any improvement. Could you give some advice? Thanks.

**Comment by xinntao** [comment]
We first train the preblur part separately (using another model only for reconstructing the clean image).  Then in the large EDVR model, we first initialize the predeblur weights with the pretrained one and set X2 smaller learning rate of the predeblur module. And then train the whole model. 

I think it is not the optimal strategy. Having another supervision like reconstructing a clean image is reasonable. I think the weight of this deblur loss term should anneal as the training goes. Did you try it?

**Comment by gulzainali98** [comment]
hey, @xinntao 
can you please help me out here with how this would work with basicsr framework?
